src/main/java/com/ericsson/ei/subscriptionhandler/InformSubscription.java

```diff
@@ -16,44 +16,42 @@
 */
 package com.ericsson.ei.subscriptionhandler;
 
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.List;
-
-import javax.annotation.PostConstruct;
-import javax.mail.MessagingException;
-
-import com.ericsson.ei.exception.SubscriptionValidationException;
 import com.ericsson.ei.jmespath.JmesPathInterface;
+import com.ericsson.ei.mongodbhandler.MongoDBHandler;
+import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.mongodb.BasicDBObject;
+import com.mongodb.util.JSON;
 import lombok.Getter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Component;
-
-import com.ericsson.ei.mongodbhandler.MongoDBHandler;
-import com.fasterxml.jackson.databind.JsonNode;
-import com.mongodb.BasicDBObject;
-import com.mongodb.util.JSON;
 import org.springframework.util.LinkedMultiValueMap;
 import org.springframework.util.MultiValueMap;
 
+import javax.annotation.PostConstruct;
+import javax.mail.MessagingException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 /**
  * This class represents the REST POST notification mechanism and the alternate
  * way to save the aggregatedObject details in the database when the
  * notification fails.
- * 
+ *
  * @author xjibbal
- * 
  */
 
 @Component
 public class InformSubscription {
 
+    private static final Logger LOGGER = (Logger) LoggerFactory.getLogger(InformSubscription.class);
+    private static final String REGEX = "^\"|\"$";
```

Comment:
better to have a single line comment on REGEX for better understand.

Reply:
just add a blank line?

Reply:
not blank line, 
// <info about "^\"|\"$" >